### PR TITLE
feat(Plugin): Add AlwaysTrust

### PR DESCRIPTION
**Pull request by zt64**

### CODE REVIEW

**Review by mantikafasi — APPROVED** [review]
Merge

File: src/plugins/alwaysTrust.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            find: ".displayName=\"MaskedLinkStore\"",
+            replacement: {
+                match: /\.isTrustedDomain=function\(.\){return .+?};/,
+                replace: ".isTrustedDomain=function(){return true};"
+            }
+        },
+        {
+            find: "\"github.com\":new RegExp(\"\\\\/releases\\\\S*\\\\/download\"),",
+            replacement: {
+                match: /const o=JSON.parse\('\[.+'\)/,
```
Comment (Vendicated) [inline src/plugins/alwaysTrust.ts:37]:
```suggestion
                match: /const o=JSON.parse\('\[.+?'\)/,
```

File: src/plugins/alwaysTrust.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+                replace: "const o=[]"
+            }
+        }
+    ]
+});
```
Comment (xafn) [inline src/plugins/alwaysTrust.ts:42]:
```suggestion
```

**Review by xafn — CHANGES_REQUESTED** [review]

File: src/plugins/alwaysTrust.ts (hunk 2 above)
Comment (Vendicated) [reply src/plugins/alwaysTrust.ts:42]: HORROR

File: src/plugins/alwaysTrust.ts (hunk 1 above)
Comment (zt64) [reply src/plugins/alwaysTrust.ts]: Whats the `?` do

File: src/plugins/alwaysTrust.ts (hunk 2 above)
Comment (zt64) [reply src/plugins/alwaysTrust.ts]: insane

File: src/plugins/alwaysTrust.ts (hunk 1 above)
Comment (Vendicated) [reply src/plugins/alwaysTrust.ts]: .+ means match as much as possible which will absolutely explode module, .+? makes it lazy, aka match as little as possible

Comment (Vendicated) [reply src/plugins/alwaysTrust.ts]: image

File: src/plugins/alwaysTrust.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+export default definePlugin({
+    name: "AlwaysTrust",
+    description: "Removes the annoying untrusted domain and suspicious file popup",
+    authors: [Devs.zt],
+    patches: [
+        {
+            find: ".displayName=\"MaskedLinkStore\"",
+            replacement: {
+                match: /\.isTrustedDomain=function\(.\){return .+?};/,
```
Comment (Vendicated) [inline src/plugins/alwaysTrust.ts:30]:
```suggestion
                match: /\.isTrustedDomain=function\(.\){return.+?};/,
```

**Pull request merged**